usagov/vote-gov-drupal
# VOTE-2002: Add state guide field to voter guide content type

## Jira ticket

[VOTE-2002](https://cm-jira.usa.gov/browse/VOTE-2002)

## Description

Add a new field for state guidance to voter guide content type.

## Deployment and testing

### Post-deploy steps

1. Run  `lando retune`

### QA/Testing instructions

1. Login as an administrator
2. Visit /admin/structure/paragraphs_type, click on the paragraph type, Double Deck Card and review the associated fields and confirm structure/configuration.
3. Visit /admin/structure/types/manage/voter_guide/fields and review structure/configuration of the new field, State Guides.
4. Edit a pre-existing Voter Guide, add test content for the new State Guides field and view the page after submitting.
5. Confirm that the field content is displaying.

## Checklist for the Developer




## Checklist for the Peer Reviewers
